release/release.sh

```diff
@@ -3,6 +3,7 @@ set -ex
 
 # shellcheck source=/dev/null
 source "$KOKORO_GFILE_DIR/common.sh"
-"$KOKORO_GFILE_DIR/verify-commits.sh"
+sudo chmod +x "$KOKORO_GFILE_DIR"/verify-commits.sh
```

Comment:
should this be `"$KOKORO_GFILE_DIR/verify-commits.sh"`? quotes seem a little weird

Reply:
I was trying a couple of things to make sure the error I'm getting isn't related to something silly like that. I'll try the chmod alone first and mess with the quotes if it's still an issue. They're effective the same thing though.